src/CoreFx.Private.TestUtilities/src/System/Diagnostics/RemoteExecutorTestBase.Process.cs

```diff
@@ -58,8 +58,16 @@ private static RemoteInvokeHandle RemoteInvoke(MethodInfo method, string[] args,
             if (!File.Exists(HostRunner))
                 throw new IOException($"{HostRunner} test app isn't present in the test runtime directory.");
 
-            psi.FileName = HostRunner;
-            psi.Arguments = testConsoleAppArgs;
+            if (options.RunAsSudo)
```

Comment:
what will happen on Windows if I set `RunAsSudo` to true? I guess it will try to find an executable called sudo and will fail?